src/cluster_slot_stats.c

```diff
@@ -4,14 +4,14 @@
  * SPDX-License-Identifier: BSD 3-Clause
  */
 
-#include "server.h"
-#include "cluster.h"
+#include "cluster_slot_stats.h"
 
 #define UNASSIGNED_SLOT 0
 
 typedef enum {
-    KEY_COUNT,
     INVALID,
```

Comment:
We can have total stat type `SLOT_STAT_COUNT` as the last member in the enum and use it in `addReplyMapLen` down below.